scripts/setup-macos.sh

```diff
@@ -58,7 +58,10 @@ function prompt {
 
 function update_brew {
   BREW_PATH=/usr/local/bin/brew
-  if [ `arch` == "arm64" ] ;
+  if [ `arch` == "arm64" ] && [ -f "$HOME/homebrew/bin/brew" ] ;
```

Comment:
Wonder if it will be better to this logic to determine default brew, but allow for a user override. 
so determine default brew like this, then do : 
BREW_PATH=${BREW_PATH:-$DEFAULT_BREW}